src/ai/backend/manager/models/gql_models/image.py

```diff
@@ -680,15 +681,16 @@ async def mutate(
         root: Any,
         info: graphene.ResolveInfo,
         registry: Optional[str] = None,
+        project: Optional[str] = None,
     ) -> RescanImages:
         log.info(
             "rescanning docker registry {0} by API request",
-            f"({registry})" if registry else "(all)",
+            f"(registry: {registry or "all"}, project: {project or "all"})",
```

Comment:
Isn't this format string broken? @jopemachine 
It looks like you have a double quote inside a double quote. 🤔

Reply:
It isn't broken in python 3.12.

See https://stackoverflow.com/questions/78388333/nested-quotes-in-f-string-with-python-3-12-vs-older-versions for details.

```
2024-12-23 04:09:02.567 INFO ai.backend.manager.models.gql_models.image [354079] rescanning docker registry (registry: all, project: all) by API request
```

However, to ensure it works correctly regardless of the Python version, it might be better to change it to single quotes, so I will replace it with single quotes